src/setupTests.js

```diff
@@ -3,3 +3,22 @@
 // expect(element).toHaveTextContent(/react/i)
 // learn more: https://github.com/testing-library/jest-dom
 import '@testing-library/jest-dom';
+import { setupServer } from 'msw/node';
+import { rest } from 'msw';
+
```

Comment:
- [ ] Kindly as it's requested on the requirements. Add tests for all your components and functions, remember that testing our applications give us scalability and stability in our code and future features or changes 🥇